lib/views/widgets/rich_text_keyboard_toolbar.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:rich_text_composer/views/commons/image_paths.dart';
+
+typedef IconWebCallback = void Function();
```

Comment:
What is this? As I understand, we only support mobile/tablet. Why related to `web`